crates/katana/core/src/backend/mod.rs

```diff
@@ -95,7 +95,7 @@ impl Backend {
             );
 
             let blockchain = Blockchain::new_from_forked(
-                ForkedProvider::new(provider, forked_block_num.into()),
+                ForkedProvider::new(provider, forked_block_num.into()).unwrap(),
```

Comment:
Would just add `expect("ForkedProvider backend thread couldn't be started")` or perhaps in next PRs with checking which `unwrap` are controlled and which are not?

Reply:
Good idea. I could put that in, but I want to refactor the caller function to also propagate the error.